src/sentry/integrations/bitbucket/integration.py

```diff
@@ -43,14 +44,30 @@ def get_client(self):
     def username(self):
         return self.model.name
 
+    def get_repositories(self):
+        repos = self.get_client().get_repos(self.username)['values']
+        data = []
+        for repo in repos:
+            data.append(
+                {
+                    'search_key': repo['name'],
```

Comment:
I actually think `search_key` isn't something needed in that UI component. Also we should try to avoid UI field binding this low in the integration.

Instaed I think the `get_repositories` interface should probably be something like `{'key': ..., 'name': ....}`